src/views/activity/Activity.vue

```diff
@@ -65,12 +65,12 @@ div
     div.row
       div.col-lg-6
         b-form-checkbox(v-model="filterAFK")
-          | Filter AFK
+          | Exclude time away from computer
```

Comment:
As mentioned in another comment, 'time away from computer' isn't something we actually know, while 'AFK' is clearly a function of the AFK-specific watcher, so I don't like this rewording.

I think we should probably avoid 'computer' here too, for mobile's sake (while they are technically computers, you don't expect an app to call the device it runs on a computer). 'device' is better, if really needed.

Reply:
Resolved in https://github.com/ActivityWatch/aw-webui/pull/291 by adding tooltips